niri-ipc/src/lib.rs

```diff
@@ -148,9 +148,9 @@ pub enum Action {
     FocusWorkspaceUp,
     /// Focus a workspace by index.
     FocusWorkspace {
-        /// Index of the workspace to focus.
+        /// Reference (index or name) of the workspace to focus.
         #[cfg_attr(feature = "clap", arg())]
-        index: u8,
+        reference: WorkspaceReferenceArg,
```

Comment:
Bikeshedding, but maybe a better name for these is `workspace`?

Reply:
Maybe? The reason I used `reference` is because of the documentation.

```
Usage: niri msg action focus-workspace <WORKSPACE>
```

...feels weird to me. Too much "workspace". "reference" is, I think, a bit more descriptive. But this is, indeed, bikeshedding, I'm happy to change it to `workspace` too.

I *will* need to update the comment above this, though, because that still says "by index" (same for all the other actions that used to work by index).